src/lib/php/Application/LicenseCsvExport.php

```diff
@@ -65,12 +65,16 @@ public function setEnclosure($enclosure='"')
    * @param int $rf Set the license ID to get only one license, set 0 to get all
    * @return string csv
    */
-  public function createCsv($rf=0)
+  public function createCsv($rf=0, $allCandidates=0)
   {
+    $forAllCandidates = "WHERE marydone = true";
+    if (!empty($allCandidates)) {
```

Comment:
Can it be `boolean` instead? It will be easier to handle then.

Reply:
Agreed